tests/python/contrib/test_cmsisnn/test_softmax.py

```diff
@@ -0,0 +1,197 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""CMSIS-NN integration tests: softmax"""
+
+import sys
+import os
+import pathlib
+import tvm
+from tvm import relay
+from tvm.relay.op.contrib import cmsisnn
+import numpy as np
+import pytest
+
+# AOT test runner is required for running CMSIS-NN tests
+# current file path: tests/python/contrib/test_cmsisnn
+# AOT Test runner: tests/python/relay/aot
+aot_tests_path = os.path.join(str(pathlib.Path(__file__).parent.resolve()), "../../relay")
+sys.path.insert(0, aot_tests_path)
+import aot
```

Comment:
Can you not use `from tests.python.relay.aot.aot_test_utils import ...` ?

Reply:
ACK